cmd/sidecars/network-passt-binding/domain/configurator.go

```diff
@@ -139,6 +141,15 @@ func (p PasstNetworkConfigurator) generateInterface() (*domainschema.Interface,
 		acpi = &domainschema.ACPI{Index: uint(p.vmiSpecIface.ACPIIndex)}
 	}
 
+	defaultGatwayLinks, err := link.DiscoverByDefaultGateway(vishnetlink.FAMILY_ALL)
+	if err != nil {
+		return err
+	}
+
+	if len(defaultGatwayLinks) != 1 {
+		return nil, fmt.Errorf("unexpected number of default gw links")
+	}
+
```

Comment:
It seems `DiscoverByDefaultGateway` can be simplified in a way it return the first matched link, WDYT?